servicex/values.yaml

```diff
@@ -41,6 +41,12 @@ app:
   mailgunApiKey:
   mailgunDomain:
 
+  # If user provides DID that lacks a schema, route the requests to this
+  # particular DID Finder
+  defaultDIDFinderScheme: rucio
+  allowedDIDSchemes:
+    - rucio
```

Comment:
Do we need to validate that `defaultDIDFinderScheme ∈ allowedDIDSchemes`? Or should that be done in the Flask app? The only tool I am aware of that will do it on a Helm chart is [`helm lint`](https://helm.sh/docs/helm/helm_lint/).

Reply:
Added a startup check to the app